### ISSUE TITLE
Support Weather Data

### ISSUE THREAD

**Issue by cglusky** [goodrobots/maverick-web#24]
Possibly with a radar overlay.

This is a slick little weather app that gets the point across:

https://play.google.com/store/apps/details?id=com.enzuredigital.weatherbomb&hl=en

**Comment by fnoop** [comment]
Yeah absolutely.  When I want to go fly, the first thing I check is the weather and I keep watching it - particularly wind and precipitation.

**Comment by cglusky** [comment]
Agree.  Just breaking out a few feature requests so we can keep an eye on The List a bit better.

If you do not have that android app I do recommend checking it out. Mostly for the swipe to manipulate time line idea.  I like it also because he has included pressure changes which is good for fishing.

**Comment by brizey02** [comment]
I have nexrad tiles baked into mavcesium, getting windspeeds and other TMS sets shouldn't be a problem.

**Comment by brizey02** [comment]
RADAR 5min updates
    layer = imageryLayers.addImageryProvider(new Cesium.WebMapServiceImageryProvider({ 
        url : '//mesonet.agron.iastate.edu/cgi-bin/wms/nexrad/n0r.cgi?', 
        layers : 'nexrad-n0r', 
        parameters : { 
            transparent : true, 
            format : 'image/png', 
            mydata : Math.random() 
        } 
    })); 
    layer.alpha = 0.5;
